test/conformance/api/v1/revision_timeout_test.go

```diff
@@ -116,7 +117,7 @@ func TestRevisionTimeout(t *testing.T) {
 
 			if tc.shouldScaleTo0 {
 				t.Log("Waiting to scale down to 0")
-				if err := WaitForScaleToZero(t, resourcenames.Deployment(resources.Revision), clients); err != nil {
+				if err := shared.WaitForScaleToZero(t, resourcenames.Deployment(resources.Revision), clients); err != nil {
```

Comment:
conformance shouldn't have anything that requires scale to zero 🤔 

cc @dprotaso

Reply:
Well, this is outside of this PR — we can remove it when we change the tests.